From 752a3ee0d4d28d77d2ec1e50559c64c6da8fe90a Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Sat, 3 Nov 2018 11:27:53 +0100
Subject: [PATCH 1/2] pie-gsi tracking

commit d6e2ba07383b0096cc05a343a9ef84a5f2009a66
Merge: 71f27d63 3879b3de
Author: Treehugger Robot <treehugger-gerrit@google.com>
Date:   Sat Nov 3 07:33:28 2018 +0000

    Merge "Ensure all android.process.media apps have usesCleartextTraffic=true" into pie-gsi

commit 3879b3def064b203b4c4ca3cb4e561a60c5b4ebc
Author: Chad Brubaker <cbrubaker@google.com>
Date:   Mon Sep 17 10:23:24 2018 -0700

    Ensure all android.process.media apps have usesCleartextTraffic=true

    Change-Id: Ib90c0f75763d52744834887633f8f6503a916b61
    Merged-In: Ib90c0f75763d52744834887633f8f6503a916b61
    Fixes: 113533479
    Test: DownloadManager CTS
    (cherry picked from commit 6cce476bc9d8fdf2edb161b878ae2a0aa6977265)
    (cherry picked from commit 0cc05adeb6a85bd250b1dd49846ad0312c76c200)

commit 71f27d63968dceabe1ba61ecde5227503a33bf9d
Merge: e2ff4365 ac5f7ab5
Author: Android Merger <noreply-android-build-merger@google.com>
Date:   Wed Oct 17 04:21:48 2018 +0000

    Merge "Merge PPR2.181005.003 from pi-release-2 into pi-platform-release. am: 352a6c8e27" into pie-gsi

commit ac5f7ab5c63e194ced9f590d95f0fd428e4e7757
Merge: e2ff4365 352a6c8e
Author: Bill Rassieur <rassb@google.com>
Date:   Tue Oct 16 21:21:42 2018 -0700

    Merge PPR2.181005.003 from pi-release-2 into pi-platform-release.
    am: 352a6c8e27

    Change-Id: I10f9635b37e0a690cde7a23343bb0314d6198c4d

commit 352a6c8e274d0af974f4bcbfa2ff62a38dffe54d
Merge: 8ac25c4c 1c947739
Author: Bill Rassieur <rassb@google.com>
Date:   Wed Oct 10 17:29:06 2018 +0000

    Merge PPR2.181005.003 from pi-release-2 into pi-platform-release.

    Change-Id: Ifee1109236150ba4d42c0765cc4cd2f714ab4519
    BUG: 117431430

commit e2ff43657fd98e3d0559eed051164d0279f434af
Merge: a5ec8ba3 6c425cc4
Author: Android Merger <noreply-android-build-merger@google.com>
Date:   Sat Sep 15 05:21:01 2018 +0000

    Merge "Merge cherrypicks of [4647037, 4647038, 4647883, 4647039, 4647933, 4648530, 4648550, 4648551, 4648552, 4648553, 4646931, 4646932, 4646933, 4646934, 4648391, 4647976, 4647977, 4647978, 4647526, 4646972, 4646935, 4646936, 4646937, 4646938, 4646939, 4646940, 4646941, 4648392, 4647509, 4648630, 4648631, 4647934] into pi-release-2 am: 8ac25c4c07" into pie-gsi

commit 6c425cc4a238503a0a2a5d121079f6df50a7c7e8
Merge: a5ec8ba3 8ac25c4c
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Fri Sep 14 22:20:56 2018 -0700

    Merge cherrypicks of [4647037, 4647038, 4647883, 4647039, 4647933, 4648530, 4648550, 4648551, 4648552, 4648553, 4646931, 4646932, 4646933, 4646934, 4648391, 4647976, 4647977, 4647978, 4647526, 4646972, 4646935, 4646936, 4646937, 4646938, 4646939, 4646940, 4646941, 4648392, 4647509, 4648630, 4648631, 4647934] into pi-release-2
    am: 8ac25c4c07

    Change-Id: I1d31c27d044da1c72a6a3090b9c8cf45c0b950af

Change-Id: Ib635ac128528bec432f89de2b3a8d9d94b3e3ab0
---
 ui/AndroidManifest.xml | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/ui/AndroidManifest.xml b/ui/AndroidManifest.xml
index fa795c9..d08ec19 100644
--- a/ui/AndroidManifest.xml
+++ b/ui/AndroidManifest.xml
@@ -11,7 +11,8 @@
                  android:icon="@mipmap/ic_launcher_download"
                  android:hardwareAccelerated="true"
                  android:supportsRtl="true"
-                 android:requiredForAllUsers="true">
+                 android:requiredForAllUsers="true"
+                 android:usesCleartextTraffic="true">
 
         <activity
             android:name=".TrampolineActivity"
-- 
2.17.1

